manifest.json

```diff
@@ -415,6 +415,13 @@
         "url": "https://github.com/yui/yuicompressor/releases/download/v2.4.8",
         "files": ["yuicompressor-2.4.8.jar"],
         "targetDirPrefix": "yuicompressor-"
+      },
+      "browsermob-proxy": {
```

Comment:
@seanlip does browsermob-proxy need to be here if it's being installed in install_third_party?